### ISSUE TITLE
Given an IBM Quantum experience job id, how do you get the logical to physical layout?

### ISSUE THREAD

**Issue by vtomole** [Qiskit/qiskit#6809]
This question is a copy of https://quantumcomputing.stackexchange.com/questions/17391/qiskit-logical-to-physical-qubit-mapping. 

I have an `IBMQJob` object that I call `circuit = IBMQjob.circuits()[0]` on. The problem is that `circuit._layout` returns `None`.

### What is the expected behavior?
I should be able to call `circuit._layout.get_virtual_bits()` to get back a dictionary of the mapping of logical --> physical qubits, like `{'q0': 0, 'q1': 9, 'q2': 13, 'q3': 14}`.

**Comment by nonhermitian** [comment]
I do not think it is possible because as of right now the disassembler does attach it https://github.com/Qiskit/qiskit-terra/blob/be0ed5f7b1c3b4057af941be65a8ff84b90a16e4/qiskit/assembler/disassemble.py#L102

**Comment by jakelishman** [comment]
Closing as stale (and would be part of `qiskit-ibm-runtime` now anyway).
